hijack/tests/hijack_tests.py

```diff
@@ -1,13 +1,14 @@
-from django.test import TestCase
+from six.moves.urllib.parse import unquote
+
+from django import VERSION
 from django.core.urlresolvers import reverse
 from django.conf import settings
 from django.contrib.auth.models import User
 from django.contrib.admin.sites import AdminSite
-from six.moves.urllib.parse import unquote
-
-from django import VERSION
 
+from django.test import TestCase, override_settings
```

Comment:
use `from django.test.utils import override_settings`
this will work in django 1.4 - 1.8